File: app/src/main/java/fr/android/androidexercises/BookItemView.java
package fr.android.androidexercises;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.LinearLayout;
import android.widget.TextView;

import org.w3c.dom.Text;

public class BookItemView extends LinearLayout {

    private TextView nameTextView;
    private TextView priceTextView;

    public BookItemView(Context context) {
        this(context, null);
    }

    public BookItemView(Context context, AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public BookItemView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected void onFinishInflate() {
        super.onFinishInflate();
        this.nameTextView = findViewById(R.id.nameTextView);
        this.priceTextView = findViewById(R.id.priceTextView);
    }

    public void bindView(Book book) {
        this.nameTextView.setText(book.name);
        this.priceTextView.setText(String.valueOf(book.price));
    }
}

File: app/src/main/java/fr/android/androidexercises/BookItemView.kt
package fr.android.androidexercises

import android.content.Context
import android.util.AttributeSet
import android.widget.TextView
import android.widget.LinearLayout



/**
 * Created by romain on 11/10/17.
 */
class BookItemView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : LinearLayout(context, attrs, defStyleAttr) {

    private var nameTextView: TextView? = null
    private var priceTextView: TextView? = null

    override fun onFinishInflate() {
        super.onFinishInflate()
        this.nameTextView = findViewById(R.id.nameTextView)
        this.priceTextView = findViewById(R.id.priceTextView)
    }

    fun bindView(book: Book) {
        this.nameTextView!!.text = book.name
        this.priceTextView!!.text = book.price.toString()
    }
}